### DISCUSSION TITLE
[Feature] automatic albums creation or categorization of photos for PC/mobile screenshots.

### DISCUSSION THREAD

**Discussion by eacunha** [immich-app/immich#13491] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When I clicked on "trash", which is empty (0 items), I *somehow* saw _all_ the screenshots of all games I played there (why?! that can be a bug report?!) - On the photo you see Mortal Online 2 Screenshots, which are pretty :D:
![image](https://github.com/user-attachments/assets/500f94d5-1d13-4a11-960a-4bc8219c4494)

This gave me the idea to create some sort of automatic library creation based on the type of photo - Eg. "Screenshots", or "documents". SOMEHOW Immich is already identifying all screenshots from all games and displaying on the "trash" tab.... so it is kind of a little bit "developed" already hah :)

I imagine it is not just myself that store ALL sorts of photos & images (not just camera photos) on the same solution, and that includes some "odd" ones like game screenshots, PC screenshots, Mobile Phone Screenshots, memes, document scans, etc... 

These "odd ones" could be also hidden from the "normal" timeline, or to have different timelines based on the "type" of image. On my attached example, I have all my games screenshots, which kind of makes a very interesting and nice visualization of my online-game-timeline. Same thing for documents could be interesting. 

Another example, just so you can see that it SOMEHOW indeed detected all games (now Genshin Impact and Pokemon Go):
![image](https://github.com/user-attachments/assets/a80f7c15-76e7-4e22-8070-80e438752570)

I would not rate this very critical/high/important, but definitely a "nice to have" :D 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Sounds like a duplicate of #1673
